_includes/manuals/2.1/1.2_release_notes.md

```diff
@@ -389,6 +389,7 @@ By default it is deployed using [systemd socket activation](http://0pointer.de/b
 
 ### Deprecations
 
+* Puppetrun are being removed from Foreman core.
```

Comment:
Puppetrun feature is being removed...

Mabe add a little bit of reasoning and what the alternative is.